easybuild/easyconfigs/p/python-parasail/python-parasail-1.2.4-fosscuda-2020b.eb

```diff
@@ -12,12 +12,10 @@ source_urls = ['https://pypi.python.org/packages/source/%(nameletter)s/parasail'
 sources = ['parasail-%(version)s.tar.gz']
 checksums = ['8e6e0762924a461bb437cbb71122765c6912dace880e338bd19c7f9d12efead2']
 
-builddependencies = [
-    ('parasail', '2.4.3'),
-]
 dependencies = [
     ('Python', '3.8.6'),
     ('SciPy-bundle', '2020.11'),
+    ('parasail', '2.4.3'),
```

Comment:
Are you sure `parasail` is a runtime dep here?

If so, maybe it's better to make this a separate PR (so it stands out in the release notes...)

Reply:
yeah, it's needed..okay, ill make a new PR